docs/_docs/usage/community-usage-examples.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Community Usage Examples
+category: Usage
+chapter: 2
+order: 7
+---
+
+This page lists various usage examples of Dependency-Track and its REST API that have been contributed by the community.
+
+### Finding vulnerabilities from CISA KEV in Dependency-Track
+
+> Contributed by [JoergBruenner](https://github.com/JoergBruenner)
+
+CISA maintains a [catalog of known exploited vulnerabilities](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) (KEV). 
+The following powershell script may be used to quickly identify projects in the Dependency-Track portfolio that are 
+affected by vulnerabilities listed in KEV.
+
+```powershell
+$api_base_url = 'http://localhost:8081'
+$api_key = 'changeit'
+
+$urlCISA = 'https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json'
+$catalog = (Invoke-WebRequest -Uri $urlCISA -Method Get).content | ConvertFrom-Json
+
+$headers = @{
+    'accept' = 'application/json'
+    'X-Api-Key' = $api_key
+}
+
+foreach ($vulnerability in $catalog.vulnerabilities)
+{
+    $uri = $api_base_url + "/api/v1/vulnerability/source/NVD/vuln/" + $vulnerability.cveID + "/projects"
+    $response = ""
+
+    try
+    {
+        $response = Invoke-WebRequest -Uri $uri -Method Get -Headers $headers
+        $affected_projects = $response | ConvertFrom-Json
+
+        if ($response.StatusCode -eq 200)
+        {
+            forEach ($project in $affected_projects)
+            {
+                $vulnerability.cveID + ': ' + $project.name + " v." + $project.version + " UUID: " + $project.uuid
+            }
+        }
+    }
+    catch
+    {
+        '[ERROR]: ' + $uri + ' / ' + $vulnerability.cveID + " : " + $response
+        '[ERROR] ' + $_.Exception.Message
+        '[ERROR] ' + $_.ScriptStackTrace
+    }
+}
+```
+
+### Creating Excel reports from EPSS data
+
+> Contributed by [JoergBruenner](https://github.com/JoergBruenner)
+
+The FIRST [exploit prediction scoring system](https://www.first.org/epss/) (EPSS) can help with prioritizing remediation
```

Comment:
This paragraph should perhaps mention that EPSS does not generate findings for ALL vulnerabilities.... just for all CVEs in a published state.  Thus, there will be no EPSS for vulns that are unique to Sonatype. GHSA, etc.

Reply:
Good spot @msymons! I'll add some clarification.

Reply:
@nscuro, @msymons Great, thank you!